[2024-09-08T07:07:05.490+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-08T07:07:05.501+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: user_input_2sum.get_sum scheduled__2024-09-08T06:00:00+00:00 [queued]>
[2024-09-08T07:07:05.506+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: user_input_2sum.get_sum scheduled__2024-09-08T06:00:00+00:00 [queued]>
[2024-09-08T07:07:05.507+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 1
[2024-09-08T07:07:05.519+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): get_sum> on 2024-09-08 06:00:00+00:00
[2024-09-08T07:07:05.530+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'user_input_2sum', 'get_sum', 'scheduled__2024-09-08T06:00:00+00:00', '--job-id', '221', '--raw', '--subdir', 'DAGS_FOLDER/python_operator_dag_with_userinput.py', '--cfg-path', '/tmp/tmpao3o95dq']
[2024-09-08T07:07:05.532+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=73803) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-08T07:07:05.533+0000] {standard_task_runner.py:105} INFO - Job 221: Subtask get_sum
[2024-09-08T07:07:05.534+0000] {standard_task_runner.py:72} INFO - Started process 73804 to run task
[2024-09-08T07:07:05.564+0000] {task_command.py:467} INFO - Running <TaskInstance: user_input_2sum.get_sum scheduled__2024-09-08T06:00:00+00:00 [running]> on host 626f2e28dec2
[2024-09-08T07:07:05.624+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='abhay' AIRFLOW_CTX_DAG_ID='user_input_2sum' AIRFLOW_CTX_TASK_ID='get_sum' AIRFLOW_CTX_EXECUTION_DATE='2024-09-08T06:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-08T06:00:00+00:00'
[2024-09-08T07:07:05.626+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-08T07:07:05.640+0000] {logging_mixin.py:190} INFO - Received num1: 0, num2: 0
[2024-09-08T07:07:05.642+0000] {logging_mixin.py:190} INFO - Sum: 0
[2024-09-08T07:07:05.643+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-08T07:07:05.652+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-08T07:07:05.656+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=user_input_2sum, task_id=get_sum, run_id=scheduled__2024-09-08T06:00:00+00:00, execution_date=20240908T060000, start_date=20240908T070705, end_date=20240908T070705
[2024-09-08T07:07:05.710+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-09-08T07:07:05.723+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-08T07:07:05.726+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
